nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -108,7 +108,7 @@ NessieCatalog initCatalog(String ref) {
     newCatalog.setConf(hadoopConfig);
     newCatalog.initialize("nessie", ImmutableMap.of("ref", ref,
         CatalogProperties.URI, uri,
-        "auth_type", "NONE",
+        "auth-type", "NONE",
```

Comment:
Is this a constant used in the Nessie project itself? If so, perhaps you might consider a follow up for adding `NessieCatalogProperties` class at some point, to help make them more clear to users looking to adopt Nessie coming from the Iceberg repo itself 👍

Reply:
yep will consider this for a future follow up, thanks